iXBRLViewerPlugin/viewer/src/js/number-matcher.js

```diff
@@ -47,9 +47,9 @@ const hyphen = '[-\u2014]';
 
 // everything that we allow immediately before a match 
 // (201c = left double quote, 2014 = em dash, 201d = right double quote)
-const begin_guard = '(?:^|(?<=(?:[/\u201c\u2014(]|' + one_ws + '|' + hyphen + ')))';
+const begin_guard = '(?:^|(?<=(?:[[/\u201c\u2014(]|' + one_ws + '|' + hyphen + ')))';
```

Comment:
This is fine, but it seems clearer to put it next to the `(` at the end of the character class (and similarly next to the `)` in `end_guard`) since they're just grouping characters.  (Alternatively, pull the `(`/`)` to the front next to the `[`/`]`).